packages/core/src/core/config.ts

```diff
@@ -73,7 +73,7 @@ export class Config {
       WARTHOG_DB_SUBSCRIBERS: ['src/subscribers/**/*.ts'],
       WARTHOG_DB_SUBSCRIBERS_DIR: 'src/subscribers',
       WARTHOG_DB_SYNCHRONIZE: 'false',
-      WARTHOG_MODULE_IMPORT_PATH: 'warthog',
+      WARTHOG_MODULE_IMPORT_PATH: '@warthog/core',
```

Comment:
I think this option can be removed altogether since it's no longer dynamic.  Does lerna + yarn workspaces allow you to pick up changes in `@warthog/core` in real time as you make the changes so that they're usable in the example?